Sources/MapboxNavigation/Style.swift

```diff
@@ -254,6 +254,43 @@ open class StylableLabel: UILabel {
     }
 }
 
+/// :nodoc:
+@objc(MBStepsTableHeaderView)
+open class StepsTableHeaderView: UITableViewHeaderFooterView {
```

Comment:
Instead of adding completely new public API for this can you try to use `UIAppearance.appearanceWhenContainedIn(_:)` method to add styling?

Reply:
The issue is that the `UITableViewHeaderFooterView` is part of the `UITableView` inside the `StepsViewController`. So right now the` UIAppearance.appearanceWhenContainedIn(_:)` couldn't find and call the property change of the section header.